svg/intro/script-content.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Test: </title>
+<link rel="help" href="https://svgwg.org/svg2-draft/intro.html#AboutSVG">
+<meta name="assert" content="The same scripts can work on both HTML and SVG elements.">
+<meta name="assert" content="The script element is in the content model of all elements.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+
+<script>var total = 0;</script>
+<script>function s0() { total += 100; return s1(); }</script>
+<svg>
+  <script>function s1() { total += 20; return s2(); }</script>
+  <g>
+    <script>function s2() { return s3(); }</script>
+    <circle id="spot">
+      <script>function s3() { return s4(); }</script>
+    </circle>
+  </g>
+  <desc>
+    <script>function s4() { return s5(); }</script>
```

Comment:
Fun side note: if the document is parsed by an HTML parser, this is going to be an HTMLScriptElement, not an SVGScriptElement. But regardless, the script content should be processed the same!